cli.js

```diff
@@ -21,6 +21,7 @@ const cli = meow(`
 	Options
 	  --overwrite         Overwrite existing DMG with the same name
 	  --identity=<value>  Manually set code signing identity (automatic by default)
+	  --dmgtitle=<value>  Manually set title of DMG volume (27 character limit)
```

Comment:
```suggestion
	  --dmg-title=<value>  Manually set title of DMG volume (27 character limit)
```

And mention that the default is the app name. And one should **only** use this if the app name is longer than 27 characters